src/representations/representation_manager_soap.hh

```diff
@@ -230,8 +236,8 @@ namespace rascal {
                   // TODO(andrea) pre compute l_factor
                   double l_factor{1 / std::sqrt(2 * l + 1)};
```

Comment:
If I am not mistaken, this should be changed to 
`double l_factor{pow(std::sqrt(2*l + 1), -1)};`
In a small test I made, for small `l`, `pow( X, -1)` outperforms `1 / X` by quite a significant amount.

Reply:
good to know